package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-maplibre-gl-srcs",
-  "version": "17.4.2",
+  "version": "^17.4.2",
```

Comment:
This is the version of the library, why setting it to "^"?

Reply:
oh shoot, good catch